src/evaluator/value/impls.rs

```diff
@@ -94,8 +94,8 @@ impl std::fmt::Debug for Value<'_> {
     }
 }
 
-impl std::string::ToString for Value<'_> {
-    fn to_string(&self) -> String {
-        format!("{:#?}", self)
+impl std::fmt::Display for Value<'_> {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:#?}", self)
```

Comment:
I changed it due to this error from Clippy in CI:

<img width="717" alt="image" src="https://github.com/Stedi/jsonata-rs/assets/1835550/b7d36d11-6492-4d8f-9688-95d9f04d34b7">